src/NzbDrone.Core.Test/ParserTests/QualityParserFixture.cs

```diff
@@ -41,6 +41,14 @@ public void should_parse_ts(string title, bool proper)
             ParseAndVerifyQuality(title, Source.TELESYNC, proper, Resolution.R720p);
         }
 
+        [TestCase("Movie Name 2018 NEW PROPER 720p HD-CAM X264 HQ-CPG", true)]
+        [TestCase("Movie Name (2022) 1080p HQCAM ENG x264 AAC - QRips", false)]
+        [TestCase("Movie Name (2018) 720p Hindi HQ CAMrip x264 AAC 1.4GB", false)]
```

Comment:
This would already be captured by CamRip

is there an HQ-Cam release you can swap it for?